libs9s/s9soptions.cpp

```diff
@@ -567,7 +567,7 @@ S9sOptions::createConfigFiles()
     userFile.fprintf("#\n");
     userFile.fprintf("# os_user          = some_user\n");
     userFile.fprintf("# os_sudo_password = some_password\n");
-    userFile.fprintf("# os_key_file      = /home/some_user/.ssh/test_ssh_key\n");
```

Comment:
FYI: 'test_ssh_key' is mentioned in cmon code in 2 places:
```
find /home/s9s/s9s/ -type f -regex ".*\.\(C\|c\|H\|h\|cpp\|hpp\|inl\)"  -not -path "*/tmpDir/*" -not -path "*/.git/*" -exec grep -nH -e "test_ssh_key" {} +
/home/s9s/s9s/clustercontrol-enterprise/src/CmonCommandHandlerWorker.cpp:7268:   "ssh_user": "cmon_user", "ssh_key": "/home/cmon_user/.ssh/test_ssh_key",
/home/s9s/s9s/clustercontrol-enterprise/tests/ut_cmonjsonmessage/ut_cmonjsonmessage.cpp:136:"{\"operation\": \"createjob\", \"job\": {\"job_data\": {\"replica_sets\": [{\"members\": [{\"address\": {\"hostname\": \"192.168.33.162\", \"port\": 27017}}], \"rs\": \"TestReplicaSet2016\"}], \"mongodb_conf_template\": \"mongodb.conf.org\", \"ssh_keyfile\": \"/home/cmon_user/.ssh/test_ssh_key\", \"vendor\": \"10gen\", \"mongos_servers\": {}, \"cluster_type\": \"mongodb\", \"sudo_password\": \"cmon_user\", \"disable_firewall\": 1, \"mongodb_password\": \"Password012222\", \"ssh_user\": \"cmon_user\", \"mongodb_user\": \"my_mongo_user\", \"mongodb_version\": \"3.2\", \"disable_selinux\": 1, \"config_servers\": []}, \"command\": \"create_cluster\"}}";
/home/s9s/s9s/s9s-tools/libs9s/s9soptions.cpp:570:    userFile.fprintf("# os_key_file      = /home/some_user/.ssh/test_ssh_key\n");
```
not sure if they also should be changed...

Reply:
That is a different thing to fix.
Different ticket should take care if important.
Although unit test is secial and the other place in cmon is just a comment.

Here in s9s we generated a default s9s.conf with it for the customers.